src/foam/dao/journal/UserCommentJournalEntryDAODecorator.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.journal',
+  name: 'UserCommentJournalEntryDAODecorator',
```

Comment:
Better to put this information as properties of JournalEntry.

Reply:
The problem with that approach is that it limits you to a single comment on a journal entry. Using decoration allows us to add any number of comments if we want to.